app/views/todos/_empty_queue.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="empty-queue">
+  <h3><%= raw t("queue_types.#{queue_type}: ") %></h3>
+
+  <div class="flash-message flash-info">
+    <p>
+      <i class="fa fa-check-circle"></i>Yay! You don't have any open <%= raw t("queue_types.#{queue_type}").downcase %>!
```

Comment:
@cleicar maybe you can humanize this as well so that it reads "pull requests" instead of pull_requests? 

See: https://api.rubyonrails.org/classes/String.html#method-i-humanize